app/Http/Controllers/GroupController.php

```diff
@@ -94,161 +94,16 @@ public function network($id)
-                        UserGroups::create([
-                            'user' => Auth::user()->id,
-                            'group' => $idGroup,
-                            'status' => 1,
-                            'role' => 3,
-                        ]);
-
-                        // Notify relevant admins
-                        $notify_admins = Fixometer::usersWhoHavePreference('admin-moderate-group');
-                        Notification::send($notify_admins, new AdminModerationGroup([
-                            'group_name' => $name,
-                            'group_url' => url('/group/edit/'.$idGroup),
-                        ]));
-
-                        if (isset($_FILES) && ! empty($_FILES)) {
-                            $file = new FixometerFile;
-                            $file->upload('file', 'image', $idGroup, env('TBL_GROUPS'), false, true);
-                        }
-                    } else {
-                        $response['danger'] = 'Group could <strong>not</strong> be created. Something went wrong with the database.';
-                        \Sentry\CaptureMessage($response['danger']);
-                    }
-                } catch (QueryException $e) {
-                    $errorCode = $e->errorInfo[1];
-                    if ($errorCode == 1062) {
-                        // This is a legitimate user error, and will be displayed to them, so we don't need to log it.
-                        $response['danger'] = __('groups.duplicate', [
-                            'name' => $name,
-                        ]);
-                    } else {
-                        $response['danger'] = __('groups.database_error');
-                        \Sentry\CaptureMessage($response['danger']);
-                    }
-                }
-            } else {
-                $response['danger'] = __('groups.create_failed');
-                \Sentry\CaptureMessage($response['danger']);
-            }
-
-            if (! isset($response)) {
-                $response = null;
-            }
-
-            if (! isset($error)) {
-                $error = null;
-            }
-
-            if (! isset($_POST)) {
-                $udata = null;
-            } else {
-                $udata = $_POST;
-            }
-
-            if (is_numeric($idGroup) && $idGroup !== false) {
-                return redirect('/group/edit/'.$idGroup)->with('response', $response);
-            }
-
-            return view('group.create', [
-                'title' => 'New Group',
-                'gmaps' => true,
-                'response' => $response,
-                'error' => $error,
-                'udata' => $udata,
-                'selectedNetworkId' => $networkId,
-            ]);
-        }
-
-        return view('group.create', [
-            'title' => 'New Group',
-            'gmaps' => true,
-            'selectedNetworkId' => $networkId,
-        ]);
+        return view('group.create');
```

Comment:
Is it OK to have done away with the 'selectedNetworkId' here?  Presume it wasn't actually doing anything?

Reply:
Exactly - you can check in the old template.